libcontainer/rootfs_linux.go

```diff
@@ -1073,17 +1073,18 @@ func remount(m mountEntry, rootfs string) error {
 		if err == nil {
 			return nil
 		}
-		// Check if the source has ro flag...
+		// Check if the source has ro, nodev, noexec, nosuid, noatime, relatime,
+		// strictatime, nodiratime flag...
 		src := m.src()
 		var s unix.Statfs_t
 		if err := unix.Statfs(src, &s); err != nil {
 			return &os.PathError{Op: "statfs", Path: src, Err: err}
 		}
-		if s.Flags&unix.MS_RDONLY != unix.MS_RDONLY {
+		if s.Flags&(unix.MS_RDONLY|unix.MS_NODEV|unix.MS_NOEXEC|unix.MS_NOSUID|unix.MS_NOATIME|unix.MS_RELATIME|unix.MS_STRICTATIME|unix.MS_NODIRATIME) == 0 {
```

Comment:
Move these flags to a const, i.e.
```go
// Check if the source mount has any of these flags...
const retryFlags = unix.MS_NODEV|unix.MS_NOEXEC|unix.MS_NOSUID|unix.MS_NOATIME|unix.MS_RELATIME|unix.MS_STRICTATIME|unix.MS_NODIRATIME
...
// ... and retry the mount with these flags set.
flags |= uintptr(s.Flags & retryFlags)

Reply:
@rpluem-vf Look this one ^^^^^

Reply:
The above code is no longer part of the commit and the comment is outdated.

Reply:
I know this request change is outdated, but he said you shuold define a const var for `unix.MS_NODEV|unix.MS_NOEXEC|unix.MS_NOSUID|unix.MS_NOATIME|unix.MS_RELATIME|unix.MS_STRICTATIME|unix.MS_NODIRATIME`. I think it's unnecessary. So this can be marked resolved?